doc/source/guide/introduction/getting_started.rst

```diff
@@ -2,4 +2,288 @@
+
+A method to remedy this problem is to instead consider a regularized problem. One of the classic regularizers is `Tikhonov regularization <https://en.wikipedia.org/wiki/Tikhonov_regularization>`_ where we instead study the problem
+
+.. math::
+   
+   \min_f ||Af - g||_2 + ||Bf||_2
+
+Where :math:`B` is a regularizer. To solve this with the above solvers, we can find the first order optimality conditions
+
+.. math::
+ 
+   2 A^* (Af - g) + 2 B^* B f =0
+
+Where :math:`A^*` is the adjoint of :math:`A`. This can be rewritten on the form :math:`Ax=b`:
+
+.. math::
+
+   (A^* A + B^* B) f = A^* g
+
+We can try solving this with :math:`B` as the `IdentityOperator` using ODL:
+
+.. code-block:: python
+
+   I = odl.IdentityOperator(space)
+   reg_op = conv.adjoint * conv + 0.1 * I.adjoint * I
+   rhs = conv.adjoint(data)
+   
+   x = space.zero()
+   odl.solvers.conjugate_gradient(reg_op, x, rhs, niter=100)
+   x.show('Tikhonov identity conjugate gradient solution')
+
+.. image:: figures/getting_started_tikhonov_identity_conjugate_gradient.png
+
+Slightly better, but no major upgrade. What about letting :math:`B` be the `Gradient`?
+
+.. code-block:: python
+
+   grad = odl.Gradient(space)
+   reg_op = conv.adjoint * conv + 0.0001 * grad.adjoint * grad
+   rhs = conv.adjoint(data)
+   
+   x = space.zero()
+   odl.solvers.conjugate_gradient(reg_op, x, rhs, niter=100)
+   x.show('Tikhonov gradient conjugate gradient solution')
+
+.. image:: figures/getting_started_tikhonov_gradient_conjugate_gradient.png
+
+Perhaps a bit better, but far from excellent. Lets try more modern methods, like TV regularization. Here we want to solve the problem
+
+.. math::
+
+   \min_{0 \leq f \leq 1} ||Af - g||_2 + ||\nabla f||_1
+
+Since this is a non-differentiable problem we need more advanced solvers to solve this problem. One of the stronger solvers in ODL is the Douglas-Rachford Primal-Dual solver (`douglas_rachford_pd`) which uses :ref:`proximal_operators` to solve the optimization problem. Several examples of this and similar solvers (`forward_backward_pd`, `chambolle_pock_solver`, etc) are available in the ODL examples folder.
+
+.. code-block:: python
+
+   # Assemble all operators
+   grad = odl.Gradient(space)
+   lin_ops = [conv, grad]
+   
+   # Create proximals as needed
+   prox_cc_g = [odl.solvers.proximal_cconj_l2(space, g=data),
+                odl.solvers.proximal_cconj_l1(grad.range, lam=0.001)]
+   prox_f = odl.solvers.proximal_box_constraint(space, 0, 1)
+   
+   # Find scaling constants
+   opnorm_conv = odl.power_method_opnorm(conv, niter=10, xstart=data)
+   opnorm_grad = odl.power_method_opnorm(grad, niter=10, xstart=data)
+   sigma = [1 / opnorm_conv**2, 1 / opnorm_grad**2]
+   
+   # Solve
+   x = space.zero()
+   odl.solvers.douglas_rachford_pd(x, prox_f, prox_cc_g, lin_ops,
+                                   tau=1.0, sigma=sigma, niter=100)
+   x.show('TV douglas rachford solution')
```

Comment:
The comments here need to be a bit more extensive, otherwise it's total mumbo-jumbo to new users.